Utils/Auth.js

```diff
@@ -19,3 +19,5 @@ module.exports.verifyAdminToken = function(req, res, next) {
 		return res.json({ Token: 'Token Not Found' });
 	}
 };
+
+module.exports = {genrateToken,verifyAdminToken}
```

Comment:
Should be spaced. Spelling is not correct.

1. Correct spelling of `generateToken`
2. rename folder from `Auth` to `auth`. `Models` to `models`

module.exports = { generateToken, verifyAdminToken }

Reply:
Resolved